nimble/controller/src/ble_ll_scan.c

```diff
@@ -1773,6 +1773,17 @@ ble_ll_scan_parse_ext_hdr(struct os_mbuf *om, struct ble_mbuf_hdr *ble_hdr,
         return -1;
     }
 
+    if (BLE_MBUF_HDR_SCAN_RSP_RCV(ble_hdr)) {
+        out_evt->evt_type |= BLE_HCI_ADV_SCAN_RSP_MASK;
+    }
+
+    if (aux_data) {
+        /* Store event type */
+        aux_data->evt_type |= out_evt->evt_type;
```

Comment:
I think if you want to store, you should do `=` (unless this does something else)

Reply:
actually I should "store" it later as "directed" bit is added below in this function.